.drone.yml

```diff
@@ -10,25 +10,17 @@ pipeline:
   # build on pull requests
 
   build:
-    image: ubuntu:xenial
+    image: node:8.8-alpine
     environment:
       - LANG=en_US.UTF-8
       - LANGUAGE=en_US:en
       - LC_ALL=en_US.UTF-8
       - BASE_URL="//talks.sourced.tech/"
     commands:
-      # we don't use rvm as install scripts need to be run under bash and not sh
-      # and Drone does not support custom shells yet. https://github.com/drone/drone/issues/1409
-      - apt-get update
-      - apt-get install -y curl git gcc make ruby2.3-dev libffi-dev zlib1g-dev locales
-      - gem install bundler
-      - locale-gen en_US.UTF-8
```

Comment:
I'm locally testing the `node:8.8-alpine` image because I got some errors in Drone. I'll consider removing that env vars if the image works. Many thanks

Reply:
It worked